packages/wallet-core/CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [0.12.0](https://github.com/statechannels/statechannels/compare/@statechannels/wallet-core@0.9.12...@statechannels/wallet-core@0.12.0) (2021-02-05)
+
+
+### Bug Fixes
+
+* add nonce to channel proposals to handle out-of-sync / adverse network situations ([9f79bc4](https://github.com/statechannels/statechannels/commit/9f79bc4a556703f3cf40c3f44fba3f0d6c28ae8e))
+* make participants a required property ([86ea421](https://github.com/statechannels/statechannels/commit/86ea421552045b837906383ef04c51c76f41bbf1))
+
+
+### Features
+
+* call challenge with initial support state ([eb3c85c](https://github.com/statechannels/statechannels/commit/eb3c85cfdcb4a5df1ef91d9f8177f482ea4b327a))
```

Comment:
This seems like an unwanted side effect, right?

(why would this be a feature of `wallet-core`?)